descriptors/ModuleDescriptor-template.json

```diff
@@ -228,6 +234,11 @@
       "permissionName": "linked-data.profiles.get",
       "displayName": "Linked Data: Get the profiles for performing CRUD operations on resources",
       "description": "Get the profiles for performing CRUD operations on linked-data resources"
+    },
+    {
+      "permissionName": "linked-data.authority-assignment-check.post",
```

Comment:
this permission should be added into package.json on UI side

Reply:
@pkjacob should i create a ticket for this?

Reply:
sure